src/quo/components/dropdowns/dropdown_input/style.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo.components.dropdowns.dropdown-input.style
+  (:require
+    [quo.foundations.colors :as colors]))
+
+(def gap 8)
+(def dropdown-width 288)
```

Comment:
I think we don't want to set width. It's better to use flex and then we set margin on the screen where the component is being used.
Usually via a 'container-style' prop which is passed to the outer container

Reply:
if you mean setting flex 1 to `root-container` & `container`, then we'll have an issue with height also being increased as component will grow. I've used width 100% combined with height 40px, if that's ok